run.env

```diff
@@ -33,6 +33,9 @@ CRON_DAYS=2
 ES_ADDON_URI="http://elasticsearch:9200"
 ES_ADDON_USER=""
 ES_ADDON_PASSWORD=""
+ES_IS_HTTPS=true
+ES_ADDON_HOST=""
+ES_ADDON_PORT=1234
```

Comment:
@nicmarti I would love to put a "better" default here, but didn't found any in the ES doc (didn't searched for hours)

Tell me and I'll update